sfge/src/main/java/com/salesforce/graph/vertex/BooleanExpressionVertex.java

```diff
@@ -61,10 +61,6 @@ public String getOperator() {
         return getString(Schema.OPERATOR);
     }
 
-    public boolean isOr() {
```

Comment:
`isOperatorOr` to match the existing method?

Reply:
`isOperatorOr` already exists, hence the deletion of this redundant (and AFAICT unused) method.